src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/TakeCard.kt

```diff
@@ -0,0 +1,111 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIcon
+import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIconView
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.event.ActionEvent
+import javafx.geometry.Pos
+import javafx.scene.layout.AnchorPane
+import javafx.scene.layout.Pane
+import javafx.scene.layout.Priority
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.common.device.IAudioPlayer
+import org.wycliffeassociates.otter.jvm.app.UIColorsObject.Colors
+import tornadofx.*
+
+class TakeCard(take: Take, player: IAudioPlayer) : AnchorPane() {
+    private val badge = stackpane {
+        style {
+            backgroundColor += c(Colors["primary"])
+            backgroundRadius += box(0.px, 10.px, 0.px, 10.px)
+            padding = box(8.px)
+        }
+        val icon = MaterialDesignIconView(MaterialDesignIcon.CREATION)
+        icon.size = "18px"
```

Comment:
you could also specify the size of the icon like this

`MaterialDesignIconView(MaterialDesignIcon.CREATION, "18px")`

Reply:
I'll double check, but I think the constructor for `MateriaDesignIconView` (vs `MaterialIconView`) doesn't support size

Reply:
I'd be curious to know the answer to that actually

Reply:
You had it right. Changed to `val icon = MaterialDesignIconView(MaterialDesignIcon.CREATION, "18px")`